_includes/body-landing.html

```diff
@@ -0,0 +1,844 @@
+          <img src="/images/rocket.svg" alt="get started" />
+          <h5 class="title">Get started</h5>
+          <p>
+            Learn Docker basics and the benefits of containerizing you
+            applications.
+          </p>
+        </a>
+      </div>
+      <div class="col-xs-12 col-sm-6 col-lg-4 card-holder">
+        <a class="card" href="https://docs.docker.com/get-docker/">
+          <img
+            src="/images/download-docker.svg"
+            alt="download and install docker"
+          />
+          <h5 class="title">Download and install</h5>
+          <p>
+            Download and install Docker on your machine in a few easy steps.
+          </p>
+        </a>
+      </div>
+      <div class="col-xs-12 col-sm-6 col-lg-4 card-holder">
+        <a class="card" href="https://docs.docker.com/develop/">
+          <img src="/images/guides.svg" alt="guides" />
+          <h5 class="title">Guides</h5>
+          <p>
+            Learn how to set up your Docker environment and start containerizing
+            you applications.
+          </p>
+        </a>
+      </div>
+      <div class="col-xs-12 col-sm-6 col-lg-4 card-holder">
+        <a class="card" href="https://docs.docker.com/release-notes/">
+          <img src="/images/whats-new.svg" alt="what's new" />
+          <h5 class="title">What's new?</h5>
+          <p>
+            Learn about the cool new features, updates, and bug fixes.
+          </p>
+        </a>
+      </div>
+      <div class="col-xs-12 col-sm-6 col-lg-4 card-holder">
+        <a class="card" href="https://docs.docker.com/engine/">
+          <img src="/images/manuals.svg" alt="products manuals" />
+          <h5 class="title">Product manuals</h5>
+          <p>
+            Browse through the manuals and learn how to use Docker products.
+          </p>
+        </a>
+      </div>
+      <div class="col-xs-12 col-sm-6 col-lg-4 card-holder">
+        <a class="card" href="https://docs.docker.com/reference/">
+          <img src="/images/reference.svg" alt="reference" />
+          <h5 class="title">Reference</h5>
+          <p>
+            Browse through the through the CLI and API reference documentation.
+          </p>
+        </a>
+      </div>
+    </div>
+  </section>
+
+  <section class="container">
+    <div class="row">
+      <div class="col-xs-12 col-lg-8 card-holder">
+        <div class="cardlet container-fluid">
+          <div class="row">
+            <div class="col-xs-12 col-lg-8">
+              <h5 class="title">How do I?</h5>
+              <p>
+                Check out the following topics to learn how to build, run, and
+                deploy your applications using Docker.
+              </p>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-xs-12 col-md-6">
+              <a href="https://docs.docker.com/engine/reference/builder/"
+                >Write a Dockerfile</a
+              >
```

Comment:
these newlines look a bit awkward (some more of these below); perhaps unwrap these lines (they will be shorter after we remove `https://docs.docker.com` from the URL's)

Reply:
Yeah, my bad, my linter is used to jsx and behaves weirdly on html... I'll fix that, thank you for noticing ;)